src/hocon_schema.erl

```diff
@@ -298,6 +302,27 @@ maybe_add_desc(Mod, Name, Meta) ->
             Meta
     end.
 
+inherit_hidden(Parent, Fields) when is_map(Fields) ->
+    case Parent of
+        #{hidden := true} ->
+            Fields#{hidden => true};
+        _ ->
+            Fields
+    end.
+
+maybe_add_hidden(Meta, Mod, Name) ->
+    case erlang:function_exported(Mod, hidden, 1) of
```

Comment:
we should not need to support this.
root schema can also be defined as map

Reply:
Thanks for the explanation, remove this function.